BUILD.gn

```diff
@@ -493,3 +494,9 @@ action("audit_deps") {
     args += [ "--audit_dev_deps" ]
   }
 }
+
+group("tools") {
+  if (enable_brave_vpn) {
+    deps = [ "//brave/components/brave_vpn:vpntool" ]
```

Comment:
Mentioned in DM; but it doesn't execute this by default (at least with `Component`). I ran manually via `src/brave/vendor/depot_tools/ninja.exe -C src/out/Component vpntool`

How does `tools` group get built?

Reply:
and it would be good to build any target with `npm run build`.
Ex, `npm run build -- --build_target=vpntool`. I'll try this as a separate task.

Reply:
This does not belong here. If you need to build it separately you can do that regardless of the location. It especially does not make sense as a general "tools" imo.